aspnetcore/tutorials/razor-pages/uploading-files.md

```diff
@@ -20,6 +20,24 @@ The [Razor Pages Movie sample app](https://github.com/aspnet/Docs/tree/master/as
 
 In the steps below, you add a movie schedule file upload feature to the sample app. A movie schedule is represented by a `Schedule` class. The class includes two versions of the schedule. One version is provided to customers, `PublicSchedule`. The other version is used for company employees, `PrivateSchedule`. Each version is uploaded as a separate file. The tutorial demonstrates how to perform two file uploads from a page with a single POST to the server.
 
+## Security considerations
+
+Caution must be taken when providing users with the ability to upload files to a server. Attackers may execute [denial of service](/windows-hardware/drivers/ifs/denial-of-service) and other attacks on a system. Some of the security steps that must be taken to reduce the likelihood that an attack will succeed are:
```

Comment:
~~Some of the security steps that must be taken to reduce the likelihood that an attack will succeed are:~~
Some security steps to reduce the likelihood that an attack succeeds are:

Reply:
Let's try this one on the next round:

> Some security steps that reduce the likelihood of a successful attack are: